spec/models/conference_spec.rb

```diff
@@ -778,6 +778,33 @@
     end
   end
 
+  describe '#scheduled_event_distribution' do
+    let!(:conference) { create(:conference) }
```

Comment:
> why let! instead of let?

Just don't want to create conference in all the tests. 

> Same for the others let!

Yes we can use let at othere places.